### ISSUE TITLE
NPE just after start

### ISSUE THREAD

**Issue by agilob** [syncthing/syncthing-android#708]
```
java.lang.NullPointerException
    at com.nutomic.syncthingandroid.syncthing.RestApi.getDevices(RestApi.java:425)
    at com.nutomic.syncthingandroid.fragments.DeviceListFragment.updateList(DeviceListFragment.java:110)
    at com.nutomic.syncthingandroid.fragments.DeviceListFragment.access$000(DeviceListFragment.java:29)
    at com.nutomic.syncthingandroid.fragments.DeviceListFragment$2$1.run(DeviceListFragment.java:53)
    at android.os.Handler.handleCallback(Handler.java:615)
    at android.os.Handler.dispatchMessage(Handler.java:92)
    at android.os.Looper.loop(Looper.java:153)
    at android.app.ActivityThread.main(ActivityThread.java:5071)
    at java.lang.reflect.Method.invokeNative(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:511)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:790)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:557)
    at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
    at dalvik.system.NativeStart.main(Native Method)
```

**Comment by licaon-kter** [comment]
App version? Store where you got the app? Device? Android version?

**Comment by agilob** [comment]
App version from Fdroid, other my devices include FBSD (syncthing 0.13.7 and archLinux with 0.14)
Android 4.1.2

It started happening a few days ago, what I think is related to upgrade to 0.14 on arch and old version of syncthing in fdroid repo.

**Comment by licaon-kter** [comment]
0.14 and 0.13 are not compatible, BTW.

Exact app version is 0.7.21? You need 0.8 for syncthitg 0.14 and 0.7.xx for syncthing 0.13.

F-Droid is not yet up to date, see the othe issue.

**Comment by agilob** [comment]
>0.14 and 0.13 are not compatible, BTW

I know, this again :(

>Exact app version is 0.7.2

The same as in Fdroid, don't have device with me.

>F-Droid is not yet up to date, see the othe issue.

I know, saw the issue today.

**Comment by Nutomic** [comment]
Fixed in 972f38755a47d9611f4d83a34140840ead57d294.
